### PR TITLE
Get-EntraBetaObjectSetting

### PR DESCRIPTION
**Pull request by v-uansari**
All example is working

### CODE REVIEW

File: module/EntraBeta/customizations/Get-EntraBetaObjectSetting.ps1 [hunk 1]
```diff
@@ -7,90 +7,73 @@
...
+        $baseUri = "https://graph.microsoft.com/beta/$TargetType/$TargetObjectId/settings"
+        $params["Method"] = "GET"
+        $params["Uri"] = $baseUri+'?$select=*'
+
         if($null -ne $PSBoundParameters["Property"])
         {
             $selectProperties = $PSBoundParameters["Property"]
             $selectProperties = $selectProperties -Join ','
-            $propertiesQuery = "`$select=$($selectProperties)"
+            $params["Uri"] = $baseUri+"`$select=$($selectProperties)"
```
Comment (KenitoInc) [inline module/EntraBeta/customizations/Get-EntraBetaObjectSetting.ps1:22]:
```suggestion
            $params["Uri"] = $baseUri+"?`$select=$($selectProperties)"
```

File: test/module/EntraBeta/Get-EntraBetaObjectSetting.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                templateId   = "bbbbbbbb-1111-2222-3333-cccccccccaaa"
+            }
+        )
+    }
+
+    Mock -CommandName Invoke-GraphRequest -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra.Beta
+}
+
+Describe "Get-EntraBetaObjectSetting" {
+    Context "Test for Get-EntraBetaObjectSetting" {
```
Comment (KenitoInc) [inline test/module/EntraBeta/Get-EntraBetaObjectSetting.Tests.ps1:26]: Add test for `Get-EntraBetaObjectSetting -TargetType "Groups" -TargetObjectId "9b424169-7f2b-4747-aa64-afbecbb28111" -Property DisplayName`

**Review by KenitoInc — CHANGES_REQUESTED** [review]

File: module/EntraBeta/customizations/Get-EntraBetaObjectSetting.ps1 (hunk 1 above)
Comment (v-pughosh) [reply module/EntraBeta/customizations/Get-EntraBetaObjectSetting.ps1]: fixed

File: test/module/EntraBeta/Get-EntraBetaObjectSetting.Tests.ps1 (hunk 2 above)
Comment (v-pughosh) [reply test/module/EntraBeta/Get-EntraBetaObjectSetting.Tests.ps1:26]: added

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**